homeassistant/components/smart_meter_b_route/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "domain": "smart_meter_b_route",
+  "name": "Smart Meter B Route",
+  "codeowners": ["@SeraphicRav"],
+  "config_flow": true,
+  "dependencies": ["usb"],
+  "documentation": "https://www.home-assistant.io/integrations/smart_meter_b_route",
+  "integration_type": "device",
+  "iot_class": "local_polling",
+  "loggers": [
+    "momonga.momonga",
+    "momonga.momonga_session_manager",
+    "momonga.sk_wrapper_logger"
+  ],
+  "quality_scale": "bronze",
+  "requirements": ["pyserial==3.5", "momonga==0.1.4"],
+  "usb": [
```

Comment:
The USB discovery is pretty broad, it will match any USB-serial chip produced by FTDI. Does the device have any USB descriptors populated? What does it look like with `lsusb`?

Reply:
This is the result of `lsusb`:
_Bus 003 Device 002: ID 0403:6015 Future Technology Devices International, Ltd Bridge(I2C/SPI/UART/FIFO)_

I am not knowledgeable about USB serials, if it can be more precise, I would be glad to restrict it.

They have multiple models: https://www.ratocsystems.com/products/wisun/usb-wisun/rs-wsuha/

Reply:
I checked with @puddly and that means we can't discover this with too many false positives

Reply:
Contacted the USB device maker, maybe a suggestion will be made.